### ISSUE TITLE
Google Chrome error in function 0x02FA2210

### ISSUE THREAD

**Issue by qt3m45su0najc7** [bleachbit/bleachbit#229]
Attached screen shot of error being reported.
error in function 0x02FA2210

**System info as follows:**

```
BleachBit version 1.12
GTK version 2.16.6
79b9-11e7-9760-f102341f075c.png)
local_cleaners_dir = C:\Program Files (x86)\cleaners
platform.win32_ver[1]() = 6.2.9200
platform.platform = Windows-Vista-10.0.15063
platform.version = 10.0.15063
sys.argv = ['C:\\Program Files (x86)\\BleachBit\\bleachbit.exe', '--gui', '--no-uac']
sys.executable = C:\Program Files (x86)\BleachBit\bleachbit.exe
sys.version = 2.5.4 (r254:67916, Dec 23 2008, 15:10:54) [MSC v.1310 32 bit (Intel)]
```

![bleachbiterrors2017-08-05](https://user-images.githubusercontent.com/465722/28995864-c237b3a4-79b9-11e7-8eb4-6a71ba6bbe5b.png)

**Comment by ROCKNROLLKID** [comment]
Can you please try the 1.17 beta here as I am pretty sure this issue was resolved either in 1.12 or in the latest beta. https://download.bleachbit.org/beta/1.17

**Comment by qt3m45su0najc7** [comment]
As you can see in the notes, I was running 1.12 whenthis occurred.  There's no way on God's earth I'm running a beta. I don't have the stomach to be a beta tester.  I'd like to see updates incorporate a better way to capture errors for easier reporting than a screen shot.

**Comment by az0** [comment]
I think this is fixed in BleachBit 1.17, so we can leave this open until the final version is released---probably with very few changes because the beta has been out for months.

Yes, error reporting is not ideal. You can also copy and paste the log as text instead of as an image.

**Comment by qt3m45su0najc7** [comment]
Hi Andrew, Sorry to respond to this with such a delay. Tks for ack and fix. Like bleachbit.

> You can also copy and paste the log as text instead of as an image.
Comment is there is no mechanism for displaying the log easily (click button to see log, menu item, link, whatever) so if you don't know where it resides, it takes a little sleuthing.  If there is a long and errors occurred, perhaps a dialog with an option to view/copy the log to clip board.
Tks.
;)

**Comment by ROCKNROLLKID** [comment]
After 2.0, we have plans to update to Python 3.x and GTK 3.x, which will allow us to do some changes to the GUI. Easier/better error reporting is one of them.

**Comment by az0** [comment]
qt3m45su0najc7 wrote he would not like to test beta versions, so I am retargeting to 2.2

**Comment by az0** [comment]
Please try [BleachBit version 2.0](https://www.bleachbit.org/news/bleachbit-20) (stable/final), and then please write back here whether it helped

**Comment by az0** [comment]
I am closing because there has been no response and because it is likely this was fixed in 2.0. If the issue comes back in 2.0 or later, please raise an issue. If the symptoms are the same, reopen this ticket. If not identical, open a new ticket.
